ocaml/doc/wire-protocol.md

```diff
@@ -55,7 +57,7 @@ These types are mapped onto XML-RPC types in a straight-forward manner:
   XML-RPC `<struct>`, with the key as the name of the struct.  Note that the
   `(k -> v) map` type is only valid when `k` is a `string`, `ref`, or
   `int`, and in each case the keys of the maps are stringified as
-  above. For example, the `(string -> double) map` containing the mappings
+  above. For example, the `(string -> float) map` containing the mappings
```

Comment:
I think this has always being wrong because the APi has floats not doubles, so here we need what the API has, not what it is mapped to.

Reply:
OCaml has only one floating point type with double precision, so I agree it is confusing to write `double` in an example which otherwise uses OCaml syntax with arrows.

Reply:
Maybe we should mention this float/double quirck. The value pased via the rpc is going to be a double

Reply:
It's mentioned a bit further up that for xml-rpc the API's float is mapped to double.
Also, further down I've described that for json-rpc the float is mapped to what consists an Intlit.